src/main/java/org/folio/services/domainevent/LogToDbFailureHandler.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.services.domainevent;
+
+import static org.apache.commons.lang3.exception.ExceptionUtils.getStackTrace;
+import static org.folio.rest.persist.PgUtil.postgresClient;
+
+import io.vertx.core.Context;
+import io.vertx.kafka.client.producer.KafkaProducerRecord;
+import java.util.Date;
+import java.util.Map;
+import java.util.UUID;
+import org.folio.persist.NotificationSendingErrorRepository;
+import org.folio.persist.entity.NotificationSendingError;
+
+public class LogToDbFailureHandler implements FailureHandler {
+  private final NotificationSendingErrorRepository repository;
+
+  public LogToDbFailureHandler(NotificationSendingErrorRepository repository) {
```

Comment:
Looks like `public` modifier can be omitted to reduce visibility.

Reply:
Changed, thanks